doc/runtime.md

```diff
@@ -31,179 +31,17 @@ Optional services:
 
 ### Using CTL's `runtime` overlay
 
-CTL's `overlays.runtime` (contained in its flake `outputs`) provides some mechanisms for conveniently launching all runtime services using [Arion](https://docs.hercules-ci.com/arion) (itself a wrapper around `docker-compose`). To use this, you must have a setup based on Nix flakes (recommended as well for [using CTL as a dependency for Purescript projects](./ctl-as-dependency.md)).
-
-Here is an example that uses the `runtime` overlay to launch all of the required services:
-
-```nix
-{
-
-  inputs = {
-    # You should probably pin this to a specific revision, especially if using
-    # it for Purescript projects
-    cardano-transaction-lib.url = "github:Plutonomicon/cardano-transaction-lib";
-
-    # To use the same version of `nixpkgs` as we do
-    nixpkgs.follows = "cardano-transaction-lib/nixpkgs";
-  };
-
-  outputs = { self, cardano-transaction-lib, nixpkgs, ... }:
-    # some boilerplate
-    let
-      defaultSystems = [
-        "x86_64-linux"
-        "x86_64-darwin"
-        "aarch64-linux"
-        "aarch64-darwin"
-      ];
-      perSystem = nixpkgs.lib.genAttrs defaultSystems;
-
-      # generate `pkgs` with CTL's overlays applied. This gives you access to
-      # various additional packages, using the same versions of CTL
-      nixpkgsFor = system: import nixpkgs {
-        inherit system;
-        overlays = [
-          cardano-transaction-lib.overlays.runtime
-          # This one is optional. If you set `ctl-server.enable = true;`
-          # in the runtime config as shown below, you must enable this
-          # overlay. `ctl-server` itself is **only** required when using
-          # CTL's `applyArgs` effect
-          cardano-transaction-lib.overlays.ctl-server
-          # you probably want this one too, although it's not demonstrated
-          # in this example
-          cardano-transaction-lib.overlays.purescript
-        ];
-      };
-
-      # The configuration for the CTL runtime, which will be passed to the
-      # expression that builds the JSON file used by Arion. This value can be
-      # shared between `buildCtlRuntime` and `launchCtlRuntime`, as shown below
-      #
-      # You can refer to the final configuration value by passing a function
-      # that takes a single arugment. Alternatively, you can pass an attrset
-      # directly
```

Comment:
The template should contain instructions on how to add services, with code examples commented-out.

Reply:
I added the instructions for the `extraServices` and `extraDockerCompose`. is that enough or should I also add also all the other services where we have default values?

Reply:
sorry, just realized that I must include all services, as otherwise the attrset isn't complete..